src/odinapi/views/read_mipas.py

```diff
@@ -22,36 +25,41 @@ def read_mipas_file(
     ifile = mipas_datapath + file
 
     data = dict()
-    with Dataset(ifile, "r") as fgr:
-        time = fgr.variables["time"][file_index]
-        if fgr.variables["time"].units == "days since 1970-1-1 0:0:0":
-            mjd_i = num2date(time, units=fgr.variables["time"].units) - datetime(
-                1858, 11, 17
-            )
-            sec_per_day = 24 * 60 * 60.0
-            data["MJD"] = mjd_i.total_seconds() / sec_per_day
-        elif fgr.variables["time"].units == "julian days":
-            data["MJD"] = time - 2400000.5
+    with tempfile.NamedTemporaryFile(suffix=".nc") as tmp:
+        buffer = s3_fileobject(f"s3://odin-vds-data/{ifile}")
+        if buffer:
+            tmp.write(buffer.read())
+
+        with Dataset(tmp.name) as fgr:
+            time = fgr.variables["time"][file_index]
+            if fgr.variables["time"].units == "days since 1970-1-1 0:0:0":
+                mjd_i = num2date(time, units=fgr.variables["time"].units) - datetime(
+                    1858, 11, 17
+                )
+                sec_per_day = 24 * 60 * 60.0
```

Comment:
Both `sec_per_day` och `datetime(1858, 11, 17)` seem like they would be neater as constants, defined in a module scope, rather than being re-initialised each call.

Reply:
didn't want to change too much, this PR was big enough already...

Reply:
Understood. Up to you if you want to do something about it.